CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contribute
+
+## Introduction
+
+First, thank you for considering contributing to rollup! It's people like you that make the open source community such a great community! 😊
+
+We welcome any type of contribution, not only code. You can help with 
+- **QA**: file bug reports, the more details you can give the better (e.g. screenshots with the console open)
```

Comment:
Maybe
"file bug reports, the more details you can give the better (e.g. [REPL](https://rollupjs.org/repl)-links or repos that demonstrate the specific issue)"
which is more helpful for the kind of issues people usually have with rollup :)